Core/src/IfcGeometryConverter/PlacementConverter.h

```diff
@@ -165,12 +164,12 @@ namespace OpenInfraPlatform {
 				 */
 				carve::geom::vector<3> convertIfcPointOnCurve(
 					const EXPRESSReference<typename IfcEntityTypesT::IfcPointOnCurve>& pointOnCurve
-				) const throw(...) {
+				) const noexcept(false) {
 					std::shared_ptr<PlacementConverterT<IfcEntityTypesT>> placementConverter
 						= std::make_shared<PlacementConverterT<IfcEntityTypesT>>(this->GeomSettings(), this->UnitConvert());
 
-					CurveConverterT<IfcEntityTypesT> gridConv(this->GeomSettings(), this->UnitConvert(), placementConverter);
-					return gridConv.getPointOnCurve<typename IfcEntityTypesT::IfcCurve>(pointOnCurve->BasisCurve, pointOnCurve->PointParameter);
+					CurveConverterT<IfcEntityTypesT> getPoint(this->GeomSettings(), this->UnitConvert(), placementConverter);
+					return getPoint.getPointOnCurve<typename IfcEntityTypesT::IfcCurve>(pointOnCurve->BasisCurve, pointOnCurve->PointParameter);

```

Comment:
Not really a better choice. I'd propose `curveConv` since it is a curve converter :)